pkgs/games/ultrastardx/default.nix

```diff
@@ -29,14 +29,14 @@ let
     sqlite lua zlib libX11 libGLU libGL ffmpeg_4
   ];
 
-in stdenv.mkDerivation rec {
+in stdenv.mkDerivation {
```

Comment:
this doesn't work without the `rec`. how did you test this?

also please align the PR title and commits/messages with contributing.md

Reply:
It appears to be working without the `rec`, it was just there already

Reply:
no https://gist.github.com/GrahamcOfBorg/f703e28dd6f27239c53d59c9a7241bbe